### PR TITLE
Add Towels

### PR DESCRIPTION
**Pull request by IProduceWidgets**
## About the PR

image

Department color towels are also included as department/role long-term playtime trinkets. 

## Why / Balance

_A towel has immense psychological value. For some reason, if a strag (strag: nonhitchhiker) discovers that a hitchhiker has his towel with him, he will automatically assume that he is also in possession of a toothbrush, washcloth, soap, tin of biscuits, flask, compass, map, ball of string, gnat spray, wet-weather gear, space suit etc., etc. Furthermore, the strag will then happily lend the hitchhiker any of these or a dozen other items that the hitchhiker might accidentally have "lost". What the strag will think is that any man who can hitch the length and breadth of the galaxy, rough it, slum it, struggle against terrible odds, win through, and still knows where his towel is, is clearly a man to be reckoned with._

## Requirements

- [x] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [x] I have added media to this PR or it does not require an ingame showcase.

## Breaking changes

**Changelog**

- add: Towels may now be brought with experienced crewmembers.

### CODE REVIEW

**Comment by Djungelskog2** [comment]
you should make them be able to be worn on your head

**Comment by Djungelskog2** [comment]
nevermind im fucking blind

**Comment by IProduceWidgets** [comment]
> you should make them be able to be worn on your head

Did do. Unfortunately due to a limitation I cant make them hide hair however.

File: Resources/Prototypes/Entities/Clothing/Multiple/towel.yml [hunk 1]
```diff
@@ -0,0 +1,729 @@
+- type: entity
+  id: Towel
```
Comment (lzk228) [inline Resources/Prototypes/Entities/Clothing/Multiple/towel.yml:2]: would be better to make BaseTowel

Comment (IProduceWidgets) [reply Resources/Prototypes/Entities/Clothing/Multiple/towel.yml]: Done!

**Comment by slarticodefast** [comment]
There's a frood who really knows where his towel is.

**Comment by UbaserB** [comment]
This is so silly omg

**Comment by GoldenCan** [comment]
Wow.
Wow.

Finally someone decided to PR this. I have tried to make some sprites myself but they were rather terrible.

Being able to equip towels in so many different regions really sells the idea of their multipurposeness, but i wish these could be used like to clean up minor spills like a bar rag, or that they could be worn over the eyes like a blindfold.

Regardless, this is a fanastic PR! The sprites look great and the towel can be worn in all the places where it would be expected.

**Comment by ps3moira** [comment]
Would it be possible to wear the towel on the shoulders too?

**Comment by deltanedas** [comment]
towel from zomboid from cdda from hitchhikers guide

**Comment by slarticodefast** [comment]
> but i wish these could be used like to clean up minor spills like a bar rag

I agree and they could probably completely replace the rag, since the rag sprite looks weird #25051

**Review by Emisse — APPROVED** [review]

**Comment by Golinth** [comment]
> Would it be possible to wear the towel on the shoulders too? Using the neck slot

Seconding this. I wanna drape it across my shoulders like the pic

**Comment by slarticodefast** [comment]
Tested in-game and everything works as intended. Should be good to merge.

**Review by slarticodefast — APPROVED** [review]

File: Resources/Prototypes/Entities/Clothing/Multiple/towel.yml [hunk 2]
```diff
@@ -0,0 +1,764 @@
...
+  parent: BaseTowel
+  components:
+  - type: Sprite
+    layers:
+    - state: icon
+      color: "#EAE8E8"
+  - type: Item
+    inhandVisuals:
+      left:
+      - state: inhand-right
```
Comment (chromiumboy) [inline Resources/Prototypes/Entities/Clothing/Multiple/towel.yml:50]: This should be 'inhand-left' for the left hand. This needs to be adjusted for the rest of the entries

Comment (IProduceWidgets) [reply Resources/Prototypes/Entities/Clothing/Multiple/towel.yml]: Huh, wonder how that happened.

**Comment by chromiumboy** [comment]
Awesome, looks good to me

**Review by chromiumboy — APPROVED** [review]

**Comment by slarticodefast** [comment]
We can give the high unlock times a try and see what the players think of them. If neccessary, we can always lower them later.
But I think it is fine considering it is only a visual color change. And it serves as a fun loyalty item for experienced players that also encourages you to try out other departments more.

**Comment by Everturning** [comment]
these times are too high. they're just TOWELS. why does someone need 100 hours as lawyer just for a dark red towel. why 100 hours as librarian just for a dark green towel. why 100hr for a black towel. why are these numbers all so goddamn high???

**Comment by GoodWheatley** [comment]
> these times are too high. they're just TOWELS. why does someone need 100 hours as lawyer just for a dark red towel. why 100 hours as librarian just for a dark green towel. why 100hr for a black towel. why are these numbers all so goddamn high???

They’re playtime achievements they’re not supposed to be trivial

**Comment by Everturning** [comment]
> > these times are too high. they're just TOWELS. why does someone need 100 hours as lawyer just for a dark red towel. why 100 hours as librarian just for a dark green towel. why 100hr for a black towel. why are these numbers all so goddamn high???
> 
> They’re playtime achievements they’re not supposed to be trivial

they are trivially over-timelocked. please explain why a person would devote 100 real life hours of their time playing one of the most boring jobs available just for a dark green towel. it isn't fun to do that.

**Comment by slarticodefast** [comment]
You don't have to unlock them all, you get a normal towel by default. They are just a different color. I would agree with you if it was a special item that has an in-game use, but it's only a minor visual difference here.

**Comment by Everturning** [comment]
> You don't have to unlock them all, you get a normal towel by default. They are just a different color. I would agree with you if it was a special item that has an in-game use, but it's only a minor visual difference here.

Exactly, a minor visual difference. So why should it be so grindy to get them? I can bear with the bronze silver and gold towel requirements. They have a general playtime requirement that allows for various play styles. My main point of contention is the specific job towels that require 4 days each for no reason other than "achievement". I'd cut it down to maybe 20 hours a job max?

**Comment by august-sun** [comment]
I mean, it's no different from the logic of some of the additional loadout options, which are also a timebound aspect. And yeah, that's the point, it's a reward for the dedication to a specific job, no other benefit than bragging rights.

**Pull request merged**